### PR TITLE
Check translate

### LINKED ISSUE #2757: Make sure all translations make sense
BHIMA 2.x has a ton of text, but the text doesn't always make sense.  Since we are going to Tshikaji to upgrade in a few weeks, it would be good if someone scanned the translated code looking for:
 1. Typos - some text is misspelled, french words without accents, etc.
 2. Unintelligible text - sentences that don't make any sense, or are applied wrong.
 3. Untranslated phrases

If we review our translations in this way, it will add a level of polish to the application.

I can take a look at the English translations.  @lomamech can you take a look at the French translations?

### PR DESCRIPTION
**Pull request by lomamech**
improvement(Translate Key)
- Fix and Improve translation
- Make all transactions make Sense

closes #2757

### CODE REVIEW

File: client/src/i18n/fr/cash_registry.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "CASH_REGISTRY" : {
-        "TITLE" : "Registre des payments"
+        "TITLE" : "Registre des paiments"
```
Comment (jniles) [inline client/src/i18n/fr/cash_registry.json]: Ce n'est pas "paiements"?

File: client/src/i18n/fr/depot.json [hunk 2]
```diff
@@ -8,23 +8,23 @@
...
-    "EDIT_DEPOT" : "Editer les informations d'un dépôt",
+    "EDIT_DEPOT" : "Éditer les informations d'un dépôt",
     "ENTITY" : "Dépôt",
     "LABEL" : "Dépôt",
     "LOCATION" : "Localisation",
     "MAIN" : {
       "TITLE" : "Gestion des dépôts"
     },
-    "JOIN_LOCATION" : "Joindre locatlisation",
+    "JOIN_LOCATION" : "Joindre la localisation",
```
Comment (jniles) [inline client/src/i18n/fr/depot.json:18]: 👍

File: client/src/i18n/fr/form.json [hunk 3]
```diff
@@ -680,26 +680,26 @@
...
             "ACCOUNT_CONFIGURATION" : "Sélectionner la configuration des comptes",
             "BALANCE_SECTION": "Sélectionner la section du bilan",
             "CASHBOX": "Sélectionner une caisse",
-            "COST_CENTER": "Sélectionner centre de cout",
-            "COUNTRY": "Sélectionner pays",
+            "COST_CENTER": "Sélectionnez un centre de coûts",
+            "COUNTRY": "Sélectionner le pays",
             "CREDITOR": "Sélectionner un Créditeur",
-            "CREDITOR_GROUP": "Sélectionner Groupe de créditeurs",
+            "CREDITOR_GROUP": "Sélectionner un Groupe créditeur",
```
Comment (jniles) [inline client/src/i18n/fr/form.json]: A mon avis, nous devons dire "Group Créditeur" tous en majescule, ou "group créditeur" tous en miniscule.

File: client/src/i18n/fr/posting_journal.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
   "POSTING_JOURNAL": {
-    "ADD_ROW" : "Ajouter Ligne",
+    "ADD_ROW" : "Ajouter une ligne",
     "CURRENCY":"Monnaie",
-    "TRANSACTION_CURRENCY":"Monnaie Transaction",
-    "REMOVE_ROW" : "Supprimer Ligne(s)",
+    "TRANSACTION_CURRENCY":"Monnaie de la transaction",
+    "REMOVE_ROW" : "Supprimer la(les) ligne(s)",
```
Comment (jniles) [inline client/src/i18n/fr/posting_journal.json:6]: 👍

**Review by jniles — COMMENTED** [review]
Looks great to me!

I made two comments to look at for clarity.  You are the expert here though.  Let me know what you think!

File: client/src/i18n/fr/form.json (hunk 3 above)
Comment (lomamech) [reply client/src/i18n/fr/form.json]: I preferred to use the first formulation "Groupe Créditeur"

**Comment by jniles** [comment]
Looks awesome!  Well done!

bors r+

**Pull request merged**